main/jlp/net/stream_client.cpp

```
@@ -0,0 +1,438 @@
+  if (err != 0 || !result) {
+    ESP_LOGW(TAG, "DNS failed for %s: %d", st.host.c_str(), err);
+    return -1;
+  }
+
+  int sock = socket(result->ai_family, result->ai_socktype, result->ai_protocol);
+  if (sock < 0) {
+    freeaddrinfo(result);
+    return -1;
+  }
+
+  struct timeval tv = {.tv_sec = kRecvTimeoutS, .tv_usec = 0};
```

Clang (14.0.6)

[warning] variable name 'tv' is too short, expected at least 3 characters

(readability-identifier-length)